index.html

```diff
@@ -0,0 +1,183 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+  <title>Main Page</title>
+  <link rel="icon" href = "assets/Icons/favicon.ico" type = "image/x-icon">
+
+  <!-- Style.css  -->
+  <link rel="stylesheet" href="./style.css">
+  <!-- COCOGOOSE & LATO FONT -->
+  <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+  <link href="http://fonts.cdnfonts.com/css/lato" rel="stylesheet">
+  <!-- Link to FontAwesome page -->
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css" 
+  integrity="sha384-oS3vJWv+0UjzBfQzYUhtDYW+Pj2yciDJxpsK1OYPAYjqT085Qq/1cq5FLXAZQ7Ay"
+  crossorigin="anonymous"/>
+
+                
+</head>
+<body>
+  <header>
+    <nav class="nav-bar">
+      <section class="nav-bar-section1">
+        <div><i onclick='openWindow("erik-fb")' class="fab fa-facebook-f"></i></div>
+        <div><i onclick='openWindow("lore-ig")' class="fab fa-instagram"></i></div>
+        <div>English</div>
+        <div>My Page</div>
+        <div>Logout</div>
+      </section>      
+      
+      <section class="nav-bar-section2">
+        <div class="nav-bar-logo">
+          <img src="assets/Logo1.png" alt="">
+        </div>
+        <ul class="nav-bar-options">
+          <li><a href="./about.html">About</a></li>
+          <li><a href="#">Program</a></li>
+          <li><a href="#">Join</a></li>
+          <li><a href="#">Sponsor</a></li>
+          <li><a href="#">News</a></li>
+        </ul>
+        <div class="nav-bar-button">IG Campaign</div>
+      </section>
+
+      <!-- Hamburguer icon -->
+      <section class="hamburguer-button">
+          <i id="burguer-icon" class="fas fa-bars"></i>
+      </section>
```

Comment:
- [ ] Kindly hide the mobile menu hamburger when the menu is clicked to open. Currently, when a user opens the menu, he/ she can still see the hamburger menu bars showing in the background of the menu. Kindly also fix this small error.

**This is how I am seeing it from my end when I open the menu**

![image](https://user-images.githubusercontent.com/86318284/181436064-78591322-7694-430e-b227-b3c34cf38959.png)

Reply:
**Clever! I have already disappeared the menu icon.**

I had not noticed this detail. Thanks for the observation.

This is how it looks now in the browser
-
![image](https://user-images.githubusercontent.com/106561762/181614959-6ba89a6e-8ebe-45ba-9bf9-bce6e1c58820.png)

And now my JavaScript file look like this 
-
![image](https://user-images.githubusercontent.com/106561762/181615176-c8cc8d37-b8cb-4dc9-85a3-4d99d1c2f128.png)